### PR TITLE
Fix reset

### LINKED ISSUE #682: reset filters with navigation reset
### From partner email
Filters seem to be getting stuck on in display. For example, at locations page, do a filter on Government (i.e. https://openaq-staging.org/#/locations?entity=Government), then reload locations page from the top menu. URL is correct (https://openaq-staging.org/#/locations) but Government filter is still turned on and non-govt data is showing up 
image001

### PR DESCRIPTION
**Pull request by jvntf**
closes #682

### CODE REVIEW

File: app/assets/scripts/main.js [hunk 1]
```diff
@@ -92,7 +92,10 @@ render(
           <Route
             name="locationsHub"
             path="/locations"
-            component={LocationsHub}
+            //component={LocationsHub}
+            component={props => (
+              <LocationsHub timestamp={new Date().toString()} {...props} />
+            )}
```
Comment (necoline) [inline app/assets/scripts/main.js:98]: What is this for?

**Comment by rwegener2** [comment]
This seems to be working great to me, but in review I realized that I can't actually reproduce this issue on staging.  @jvntf you were able to reproduce the issue, I take it?

File: app/assets/scripts/main.js (hunk 1 above)
Comment (jvntf) [reply app/assets/scripts/main.js:98]: oops first solution idea, will remove

**Comment by jvntf** [comment]
@rwegener2 yes i was able to reproduce

**Review by rwegener2 — APPROVED** [review]

**Review by necoline — APPROVED** [review]

**Pull request merged**